### DISCUSSION TITLE
[rclone] Use appdata config instead of separate one

### DISCUSSION THREAD

**Discussion by EpicLPer** [ScoopInstaller/Main#5053] [Ideas]
Hi,

Is it somehow possible to use the global rclone config located in "AppData\Roaming\rclone\rclone.conf" instead of its own Scoop app path?
I've been trying to figure out why rclone wouldn't take my updated config file (used WinGet before but it somehow broke) and saw that it created its own config file in its own path.

Thanks!

**Comment by HUMORCE** [comment]
Create your own manifest instead of official bucket.
